src/index.js

```diff
@@ -1070,17 +1070,21 @@ class Offline {
   }
 
   // All done, we can listen to incomming requests
-  _listen() {
-    return new Promise((resolve, reject) => {
-      this.server.start(err => {
-        if (err) return reject(err);
+  async _listen() {
+    try {
+      await this.server.start();
+    }
+    catch (e) {
+      // TODO FIXME
+      // should we just write the error to console and exit?
+      console.error('Unexpected error in serverless-offline.', e);
+      process.exit(1);
```

Comment:
`'Unexpected error while starting serverless-offline server:'` would be more verbose.
Otherwise I think logging the error and exiting 1 is correct.
Please remove the comments